lib/tasks/mastodon.rake

```diff
@@ -33,6 +33,58 @@ namespace :mastodon do
     end
   end
 
+  module AvatarHeader
```

Comment:
I'm afraid this is really not idiomatic Ruby code and it is doing *a lot* of things not the right way, it will also not work with S3 or any other cloud storage backends due to trying to write to local files. In the example I have given you, I demonstrated how to do it using Paperclip in a really simple way